docs/sources/review/test-documentation-changes/index.md

```diff
@@ -14,14 +14,16 @@ weight: 200
 {{< docs/shared source="writers-toolkit" lookup="make-help.md" >}}
 
 To run the local documentation web server, run `make docs` from the `docs/` directory.
-The output message includes a URL that you can follow to view the changes to the documentation in the browser.
-Refer to an [example of a successful build](#example-succesful-build).
+If you are in the website repository, run `make docs` from the root of the repository instead.
 
 {{< admonition type="note" >}}
-Running `make docs` from the root of a repository produces the output `make: Nothing to be done for 'docs'.` instead of running the local documentation web server.
+Apart from in the website repository, running `make docs` from the root produces the output `make: Nothing to be done for 'docs'.` instead of running the local documentation web server.
```

Comment:
```suggestion
Running `make docs` from the wrong directory, produces the output `make: Nothing to be done for 'docs'.` or `make: *** No rule to make target 'docs'.  Stop.`, and the local documentation web server is not started.
```

Maybe make it cover the two errors you can get? And take the awkward `Apart...` structure out?

It's good as is too :-) Just thinking of the other errors users can get.

Reply:
much neater!